salt/modules/zypper.py

```diff
@@ -665,23 +665,20 @@ def install(name=None,
                        'new': '<new-version>'}}
     '''
     try:
-        pkg_params, pkg_type = __salt__['pkg_resource.parse_targets'](
-            name, pkgs, sources, **kwargs
-        )
+        pkg_params, pkg_type = __salt__['pkg_resource.parse_targets'](name, pkgs, sources, **kwargs)
```

Comment:
Yes. Basically, I just put it back. That was LISP'ified by Erik exactly moved around just to move it around. :wink: I would like to have it also less scrollable, because the method is pretty long and these kind of bugs would be better to spot next time...